challenge_3/python/CindyB/src/majorityElement.py

```diff
@@ -0,0 +1,7 @@
+def findMajorityElement(myArray):
+    for x in myArray:
+        count = sum(map(lambda s: s == x,myArray))
```

Comment:
Again, this is O(n^2). I suggest looking at the [Counter](https://docs.python.org/3/library/collections.html?highlight=counter#collections.Counter) class or [defaultdict](https://docs.python.org/3/library/collections.html?highlight=counter#collections.defaultdict)